File: Java/Class.java

class User {

    private String username;
    private String password;

    static String SEX = "BOY";

    User(String username, String password) {
        this.username = username;
        this.password = password;

        System.out.println("call User class");
    }

    @Override
    public String toString() {
        return "[ username = " + username + ", password = " + password + "]";
    }

    final String getUsername() {
        return username;
    }
}

class People extends User {

    People(String username, String password) {
        super(username, password);

        System.out.println("call people class");
    }
}

public class Class {

    public static void main(String[] args) {

        People people = new People("turaiiao", "12345678");

        System.out.println(people.toString());
        System.out.println(User.SEX);
        System.out.println(people.getUsername());
    }
}

File: Python/class.py

from enum import Enum, unique

class ErrorCode(Enum):
    UsernameIncorrect = 'username incorrect'
    PasswordIncorrect = 'password incorrect'
    
    Success = 'login successfully'

class User:
    def __init__(self, username, password):
        self.username = username
        self.password = password
        
    def validate(self, username, password):
        if self.username != username:
            return ErrorCode.UsernameIncorrect
        elif self.password != password:
            return ErrorCode.PasswordIncorrect
        else:
            return ErrorCode.Success

if __name__ == '__main__':
    a = User('turaiiao', password)
    
    username = input('please input your name: ')
    password = input('please input your pass: ')
    
    b = a.validate(username, password)
    
    print(b.value)
